### ISSUE TITLE
Error with Electron project

### ISSUE THREAD

**Issue by ArronYR** [dazoe/ed25519#17]
hi, in my project, I need use `ed25519` library. If I run the program directly after `npm install`, an error will be reported:

```
┏ Electron -------------------

  dyld: lazy symbol binding failed: Symbol not found: _SHA512
    Referenced from: /Users/arron/Documents/node_workspace/vue-project/node_modules/ed25519/build/Release/ed25519.node
    Expected in: flat namespace

  dyld: Symbol not found: _SHA512
    Referenced from: /Users/arron/Documents/node_workspace/vue-project/node_modules/ed25519/build/
┗ ----------------------------

┏ Electron -------------------

  Release/ed25519.node
    Expected in: flat namespace


┗ ----------------------------
```

- NodeJS Version: 8.0.0
- Electron Version: 1.8.1
- Operating system: macOS High Sierra 10.13

**Comment by abunsen** [comment]
+1

**Comment by KenanSulayman** [comment]
The method `SHA512` is referenced by the `ed25519` module and resides in a respective OpenSSL library that's installed on the computer.

What's the installed openssl version? (`openssl version`)
What version of Clang are you using?

**Comment by ChildishForces** [comment]
Also having this exact issue.

**OpenSSL:** 
OpenSSL 1.0.2p  14 Aug 2018
**CLang:**
Apple LLVM version 9.1.0 (clang-902.0.39.2)
Target: x86_64-apple-darwin18.0.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin

**Comment by gaoxiangxyz** [comment]
I meet this issue too. I solve it by removing dependencies on openssl. https://github.com/gaoxiangxyz/ed25519

**Comment by erikma** [comment]
I merged @gaoxiangxyz 's change 8900a825370393ec723f88f74d63d16c509343c2 containing the fix into master. Updated package coming when I get permissions.
